### PR TITLE
HTCONDOR-2235 Organize info about job runtime services into one section

### PR DESCRIPTION
**Pull request by GregThain**
Also tighten up section names to fit on one line

<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/users-manual/env-of-job.rst [hunk 1]
```diff
@@ -0,0 +1,1707 @@
...
+
+The :subcom:`container_service_names` submit command accepts a comma- or space-
+separated list of service names; each service name must have a corresponding
+:subcom:`<service-name>_container_port` submit command specifying an integer
+between 0 and 65535.  Docker will automatically select a port on the host
+to forward to that port in the container; HTCondor will report that port
+in the job ad attribute :subcom:`<service-name>_HostPort` after it becomes
+available, which will be (several seconds) after the job starts.  HTCondor
+will update the job ad in the sandbox (``.job.ad``) at that time.
+Virtual Machine Jobs
```
Comment (ColeBollig) [inline docs/users-manual/env-of-job.rst:450]:
```suggestion
will update the job ad in the sandbox (``.job.ad``) at that time.

Virtual Machine Jobs
```

File: docs/users-manual/env-of-job.rst [hunk 2]
```diff
@@ -0,0 +1,1707 @@
...
+Take care with respect to the use of network connections within the
+virtual machine and their interaction with checkpoints. Open network
+connections at the time of the checkpoint will likely be lost when the
+checkpoint is subsequently used to resume execution of the virtual
+machine. This occurs whether or not the execution resumes on the same
+machine or a different one within the HTCondor pool.
+
+Disk Images
+'''''''''''
+
```
Comment (ColeBollig) [inline docs/users-manual/env-of-job.rst:640]: Is this section supposed to be empty?

File: docs/users-manual/env-of-job.rst [hunk 3]
```diff
@@ -0,0 +1,1707 @@
...
+parallel universe provides machine scheduling, but does not enforce a
+particular programming paradigm for the underlying applications. Thus,
+parallel universe jobs may run under various MPI implementations as well
+as under other programming environments.
+
+The parallel universe supersedes the mpi universe. The mpi universe
+eventually will be removed from HTCondor.
+
+How Parallel Jobs Run
+;''''''''''''''''''''
```
Comment (ColeBollig) [inline docs/users-manual/env-of-job.rst:790]:
```suggestion
'''''''''''''''''''''
```

File: docs/users-manual/env-of-job.rst [hunk 4]
```diff
@@ -0,0 +1,1707 @@
...
+.. code-block:: condor-submit
+
+    +ParallelShutdownPolicy = "WAIT_FOR_ALL"
+
+in the submit description file. It causes HTCondor to wait until every
+node in the parallel job has completed to consider the job finished.
+
+Parallel Jobs and the Dedicated Scheduler
+;''''''''''''''''''''''''''''''''''''''''
+
```
Comment (ColeBollig) [inline docs/users-manual/env-of-job.rst:833]:
```suggestion

Parallel Jobs and the Dedicated Scheduler
'''''''''''''''''''''''''''''''''''''''''

```

File: docs/users-manual/env-of-job.rst (hunk 2 above)
Comment (GregThain) [reply docs/users-manual/env-of-job.rst:640]: Yes.

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**